UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -160,6 +160,10 @@ public partial class GameManager : MonoBehaviour, IInitialise
 
 	[NonSerialized]
 	public bool DisconnectExpected = false;
+
+	///tuto part
+    public bool onTuto;
```

Comment:
indentation seems to be wrong. You also have to make sure it follows our naming conventions and make it ``PascalCase``. If possible, make a public interface instead of a public field.